### DISCUSSION TITLE
Blacklist blocks for use of /top

### DISCUSSION THREAD

**Discussion by MCLimited** [Zrips/CMI#5067] [Feature Requests]
At the moment when you use /top you get on top of barrier roofs. This is annoying when you use barriers to avoid snow in your creative build or when you made a building they are not allowed to leave with a barrier roof. 

It would be nice to have a feature to be able to add blacklisted blocks for /top command.

For example:
Blacklist-blocks-top:
  - barrier
  - bedrock

**Comment by CloudeLecaw** [comment]
u know request here got ignored by the devs mostly? This here is jsut for issues
send them here aggain: https://github.com/Zrips/CMI/discussions
